packages/apidom-playground/vite.config.mjs

```diff
@@ -0,0 +1,26 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+
+export default defineConfig({
+  plugins: [
+    react(),
+    {
```

Comment:
This plugin is a replacement for this part of the previous `start` script:
```
cross-env GENERATE_SOURCEMAP=false
``` 
Using other sourcemap configs did not disable them for JS files. 

Source: https://github.com/vitejs/vite/issues/9825#issuecomment-2413567622